library/src/services/api/ParsedApiSpecInterface.ts

```diff
@@ -86,6 +85,7 @@ export interface ScalarProperty {
   required: boolean;
   pattern?: RegExp;
   helpText?: string | React.ReactElement<any>;
+  downloadable?: boolean;
```

Comment:
This will require the developer to activate file download feature on every single column. We could just auto-detect it and show the download button